### PR TITLE
Speed up tests and add release yaml

### PR DESCRIPTION
**Pull request by mvpatel2000**
Speeds up cpu tests

deletes mosaic_gpt yamls

adds release workflow

image
Tests arae now 4 min + overhead. I expect docker part to speed up w caching, on composer it only takes 30sec.

### CODE REVIEW

File: scripts/data_prep/convert_dataset.py [hunk 1]
```diff
@@ -114,6 +114,16 @@ def __init__(self,
         super().__init__(hf_split, folder_split, raw_samples, truncated_samples)
 
 
+class ValSmallConstants(DataSplitConstants):
```
Comment (dakinggg) [inline scripts/data_prep/convert_dataset.py:117]: this class looks unused?

**Review by dakinggg — COMMENTED** [review]
why'd you delete a bunch of yamls? am i missing something?

**Comment by abhi-mosaic** [comment]
> why'd you delete a bunch of yamls? am i missing something?

The `mosaic_gpt` folder is gone now, for some reason it was still around.

File: scripts/data_prep/convert_dataset.py (hunk 1 above)
Comment (mvpatel2000) [reply scripts/data_prep/convert_dataset.py:117]: @codestar12 ? Abhi said u might be familiar w this file, what does it do

Comment (dakinggg) [reply scripts/data_prep/convert_dataset.py:117]: did you mean for this to say `ValXSmallConstants`? (still unused, but at least would match everything else then

**Pull request merged**